src/com/intellij/plugins/haxe/util/HaxeResolveUtil.java

```diff
@@ -578,21 +578,26 @@ public static String getQName(@NotNull PsiElement type, boolean searchInSamePack
       return usingStatement.getReferenceExpression().getText();
     }
 
-    String text = type.getText();
-    if (type instanceof HaxeReferenceExpression) {
```

Comment:
Seems like it was a bad idea to add this before next part. (https://github.com/TiVo/intellij-haxe/commit/4b822108dc322edc122dfe06aa34e37b5b081253)
https://github.com/TiVo/intellij-haxe/issues/39